website/docs/configuration/interpolation.html.md

```diff
@@ -392,6 +392,11 @@ The supported built-in functions are:
 
   * `title(string)` - Returns a copy of the string with the first characters of all the words capitalized.
 
+  * `transpose(map)` - Returns map of (string,list) from a map (string,list)
```

Comment:
We usually refer to this sort of structure as "map of lists of strings". Although this is more verbose than the notation you used here, I think it's best to be consistent with other usage to maximize clarity here.

Perhaps the following, relying more on the example to illustrate the behavior rather than prose:

> Swaps the keys and list values in a map of lists of strings. For example, `transpose(map("a", list("1", "2"), "b", list("2", "3"))` produces a value equivalent to `map("1", list("a"), "2", list("a", "b"), "3", list("b"))`.

Reply:
Excellent suggestion. Thanks!